server/test/unit/server/async/test_app.py

```diff
@@ -1,11 +1,16 @@
 """
 This module contains tests for the pulp.server.async.app module.
 """
-import unittest
-
 import mock
+import unittest
+import platform
 
 from pulp.server.async import app
+from pulp.common.constants import RESOURCE_MANAGER_WORKER_NAME
```

Comment:
For organization, it helps to keep these sorted. So this should go before any ``pulp.server`` imports.